api/src/main/java/org/openmrs/CohortMembership.java

```diff
@@ -10,10 +10,14 @@
 package org.openmrs;
 
 import java.util.Date;
+import java.util.Objects;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * @since 2.1.0
+ */
```

Comment:
(copying a comment I made on the original commit)

@vshl @wluyima when I see this I wonder why CohortMembership has a "Patient patient" field instead of just an "Integer patientId". Wouldn't that more correctly capture the data that's available?

Cohort used to just have a Set; are we intentionally now using the fact that it's a Patient somewhere?
Otherwise I propose we change CohortMembership to just have the Integer patientId.

(Note that doing `new Patient(Integer)` will have a randomly-generated UUID, which could be misleading to consuming code, e.g. when I try to add this in the REST webservices module.